app/ui/client/components/header/headerRoom.js

```diff
@@ -177,6 +177,31 @@ Template.headerRoom.events({
 			});
 		}
 	},
+	'click .rc-header__content.rc-header__block'(event, instance) {
+		const { tabBar } = instance.parentTemplate();
+		const $flexTab = $('.flex-tab-container .flex-tab');
+
+		if (tabBar.getState() === 'opened' && (tabBar.getTemplate() === 'channelSettings' || tabBar.getTemplate() === 'membersList')) {
+			$flexTab.attr('template', '');
```

Comment:
Is this right setting the html property manually?

Reply:
It's been done the same way in flexTabBar.js. I think he got the snippet from there actually.

Reply:
That accurate. I was looking for how it had been implemented, and that's the file I got to.